test/e2e/framework/util.go

```diff
@@ -3795,6 +3797,13 @@ type SSHResult struct {
 	Code   int
 }
 
+// NodeExec execs the given cmd on node via SSH. Note that the nodeName is an sshable name,
+// eg: the name returned by framework.GetMasterHost(). This is also not guaranteed to work across
+// cloud providers since it involves ssh.
+func NodeExec(nodeName, cmd string) (SSHResult, error) {
+	return SSH(cmd, fmt.Sprintf("%v:%v", nodeName, sshPort), TestContext.Provider)
```

Comment:
I won't block on it, but it seems like increased surface area without a whole lot of benefit